seatunnel-common/src/main/java/org/apache/seatunnel/common/config/Common.java

```diff
@@ -70,19 +54,17 @@ public static Optional<String> getDeployMode() {
      * When running seatunnel in --master yarn or --master mesos, you can put plugins related files in plugins dir.
      */
     public static Path appRootDir() {
-        if (MODE.equals(Optional.of(DeployMode.CLIENT.getName()))) {
+        if (DeployMode.CLIENT == MODE) {
             try {
                 String path = Common.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath();
                 path = new File(path).getPath();
                 return Paths.get(path).getParent().getParent();
             } catch (URISyntaxException e) {
                 throw new RuntimeException(e);
             }
-        } else if (MODE.equals(Optional.of(DeployMode.CLUSTER.getName()))) {
-            return Paths.get("");
-        } else {
-            throw new IllegalStateException("MODE not support : " + MODE.orElse("null"));
         }
+        // deploy mode: cluster
```

Comment:
Please cover this logic in if statement, if we have new another DeployMode then throw not support exception

Reply:
done.